### ISSUE TITLE
About ProjectedGANLoss code.

### ISSUE THREAD

**Issue by higun2** [autonomousvision/stylegan-t#19]
Why divide by 'batch_size' when calculating GAN loss?

> loss_gen = (F.relu(torch.ones_like(gen_logits) + gen_logits)).mean() / batch_size
> loss_real = (F.relu(torch.ones_like(real_logits) - real_logits)).mean() / batch_size
> loss_gen = (-gen_logits).mean() / batch_size

Doesn't 'mean.()' already include dividing by batchsize?
